crates/relayer-cli/src/commands/tx/channel.rs

```diff
@@ -744,26 +746,36 @@ pub struct TxChanUpgradeInitCmd {
         value_name = "TIMEOUT_TIMESTAMP",
         help = "Timestamp that, once it has been surpassed on the originating chain, the upgrade will time out. Required if no timeout height is specified."
     )]
-    timeout_timestamp: Option<Timestamp>,
+    timeout_timestamp: u64,
```

Comment:
Why this change?

Reply:
It was to have a similar timeout as `ft-transfer` where a difference in seconds is given instead of a specific `Timestamp`. Maybe renaming it to `timeout_seconds`, same as the one in `ft-transfer`, would be better.

https://github.com/informalsystems/hermes/blob/2c65075fbd4c24c4b947a633116a46ed0a13712a/crates/relayer-cli/src/commands/tx/channel.rs#L765